packages/nouns-contracts/contracts/governance/NounsDAOLogicV2.sol

```diff
@@ -901,6 +971,14 @@ contract NounsDAOLogicV2 is NounsDAOStorageV2, NounsDAOEventsV2 {
         }
     }
 
+    function _refundGas(uint256 startGas) internal {
+        uint256 gasPrice = min(tx.gasprice, block.basefee + MAX_REFUND_PRIORITY_FEE);
+        uint256 gasUsed = startGas - gasleft() + REFUND_BASE_GAS;
+        uint256 refundAmount = min(gasPrice * gasUsed, address(this).balance);
+        (bool refundSent, ) = msg.sender.call{ value: refundAmount }('');
+        emit RefundableVote(msg.sender, refundAmount, refundSent);
```

Comment:
Looking at the calculations in this block, overflow should not be possible. How about wrapping in `unchecked` to save a few hundred gas?

Reply:
absolutely: https://github.com/nounsDAO/nouns-monorepo/pull/515/commits/2a911cf693a4c99c7aedb2c4061389bfc9d5f992